test/Microsoft.AspNetCore.Mvc.ViewFeatures.Test/ControllerTest.cs

```diff
@@ -153,6 +177,158 @@ public void Controller_View_WithParameterViewNameAndViewModel_SetsResultViewName
         }
 
         [Fact]
+        public void Controller_View_WithParameterNullViewModel_MaintainsNullModel()
+        {
+            // Arrange
+            var controller = new TestableController()
+            {
+                ViewData = new ViewDataDictionary(new EmptyModelMetadataProvider()),
+                TempData = new TempDataDictionary(new DefaultHttpContext(), Mock.Of<ITempDataProvider>()),
+            };
+
+            // Act
+            var actualViewResult = controller.View(model: null);
+
+            // Assert
+            Assert.IsType<ViewResult>(actualViewResult);
+            Assert.Null(actualViewResult.ViewName);
+            Assert.Same(controller.ViewData, actualViewResult.ViewData);
+            Assert.Same(controller.TempData, actualViewResult.TempData);
+            Assert.Null(actualViewResult.ViewData.Model);
+        }
+
+        [Fact]
+        public void Controller_PartialView_WithoutParameter_SetsResultNullViewNameAndNullViewDataModelAndSameTempData()
+        {
+            // Arrange
+            var controller = new TestableController()
+            {
+                ViewData = new ViewDataDictionary(new EmptyModelMetadataProvider()),
+                TempData = new TempDataDictionary(new DefaultHttpContext(), Mock.Of<ITempDataProvider>()),
+            };
+
+            // Act
+            var actualViewResult = controller.PartialView();
+
+            // Assert
+            Assert.IsType<PartialViewResult>(actualViewResult);
+            Assert.Null(actualViewResult.ViewName);
+            Assert.Same(controller.ViewData, actualViewResult.ViewData);
+            Assert.Same(controller.TempData, actualViewResult.TempData);
+            Assert.Null(actualViewResult.ViewData.Model);
```

Comment:
This is not a useful test. There's nothing which could possibly set `ViewData.Model`. And, you've already got `Controller_PartialView_WithoutParameter_MaintainsModelInViewData`. Just remove this one unless other checks were missing.